### ISSUE TITLE
swipe up/down sensitivity seems to be overridden by right/left many times

### ISSUE THREAD

**Issue by sysrage** [rpetrich/ProSwitcher#28]
when trying to swipe up or down in ProSwitcher to either exit proswitcher or close the app, many times it doesn't work. it seems to be caused by highly sensitive right/left swiping. i think this because sometimes the apps will start sliding slightly to the left or right. this could be tweaked so if there is more upward or downard motion than there is right/left motion, it will close app or close proswitcher instead. would make this a lot easier to use for me.. maybe i'm the only one that's not good at swiping perfectly up/down.

**Comment by grp** [comment]
Sorta hard to implement :/.

**Comment by rpetrich** [comment]
Yeah, this is really difficult to implement because scroll views steal touch events :(

Perhaps we can subclass from UIScrollView and override handlePan: ;)

**Comment by grp** [comment]
I'd say just leave it be since subclassing can lead to a non-iphoney experience, especially with UIScrollView :/

**Comment by ejoya** [comment]
@sysrage touch the app for less than 1 sec before swiping to close the app.. it's like dragging the app instead of swiping it fast. This always works for me.

**Comment by j4m1eb** [comment]
@ejoya  great tip, I was struggling with this too, but if you touch the card, hold it then flick it up it works everytime.
